### ISSUE TITLE
Remove border from buttons and images (may be links), when focussed

### ISSUE THREAD

**Issue by cjchirag7** [cjchirag7/concetto_front#45]
May be you need to change the :focus property of button or a or something else to make the outline : none, border: none
You can get help by just searching on google

**Comment by VarshaSulakhe** [comment]
I would like to work on this

**Comment by cjchirag7** [comment]
![Border on focus](https://user-images.githubusercontent.com/45796806/66257030-dc22b300-e7b1-11e9-8fd4-1690d0de7397.png)
See the rectangular border which comes just only when a user clicks a link. This border/outline shall be removed

**Comment by VarshaSulakhe** [comment]
I don't see any border on chrome. May I know which browser you are running it on?

**Comment by cjchirag7** [comment]
Firefox

**Comment by cjchirag7** [comment]
This border comes only for a short instant, when the user clicks a link....

**Comment by VarshaSulakhe** [comment]
Okay.

**Comment by VarshaSulakhe** [comment]
I am not able to reproduce the issue on Firefox as well. Do you see anything that we might be missing here?
<img width="386" alt="Screenshot 2019-10-06 at 6 53 26 PM" src="https://user-images.githubusercontent.com/17929485/66269801-a7befd80-e86a-11e9-9dbe-68c8b32e20bd.png">

**Comment by cjchirag7** [comment]
Try to **click on any link** (try the links in hamburger menu also). This shall produce a border or outline around the link

**Comment by VarshaSulakhe** [comment]
Yeah. I don't see that issue when i'm clicking on any link. None on hamburger menu as well
<img width="387" alt="Screenshot 2019-10-06 at 7 46 21 PM" src="https://user-images.githubusercontent.com/17929485/66270504-49961880-e872-11e9-86fc-eab6126317f3.png">
